### PR TITLE
Travis CI: docker image switch and minor fixes

### PR DESCRIPTION
**Pull request by bhuesemann**
Includes:
- minor gradlew fix 
- travis ci: 
 -- switch to openjdk
 -- reflect changes in syntax of the travis yml
 -- switch base docker image for db2

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,33 +1,45 @@
...
 services:
   - docker
   - postgresql
 
 language: java
+
 before_install:
-  - docker run --name dbfitdb2 -p 50000:50000 -v "`pwd`/test_vm/scripts/db2:/scripts" -e DB2INST1_PASSWORD=db2inst1-pwd -e LICENSE=accept ibmcom/db2express-c:latest db2start &
-  - docker run -p 1521:1521 -p 49160:22 -v "`pwd`/test_vm/scripts/oracle/sql:/docker-entrypoint-initdb.d:ro" wnameless/oracle-xe-11g &
+  - docker run -d -p 50000:50000 -v "`pwd`/test_vm/scripts/db2:/scripts" -e DB2INST1_PASSWORD=db2inst1-pwd -e LICENSE=accept --name dbfitdb2 --privileged=true cwds/db2 &
```
Comment (javornikolov) [inline .travis.yml:12]: Do we need, the `-d`? I think there used to be something with the level of detail logged. We anyway run them in background mode.

**Comment by javornikolov** [comment]
I general looks good. Just this curiousity about -d switch

File: .travis.yml (hunk 1 above)
Comment (bhuesemann) [reply .travis.yml:12]: During debugging I had removed running the docker images in the background and thus I've detached the containers. It's not necessary if you run it in background on the OS level. I don't know if this has influence on logging. I'll revert it.

**Comment by bhuesemann** [comment]
Should be ok now.

**Pull request merged**